### PR TITLE
Static  container refactorization

### LINKED ISSUE #2694: [Static Content] Refactor template to use .container module
Static content template is one of the last stragglers to be refactored. Update this shiznit!

### PR DESCRIPTION
**Pull request by weerd**
Resolves #2694

@angaither helped me move the header output from the `preprocess_page` to the `preprocess_node` so that we have access to it in the static content node template. This completes updating Static Content to modern times and using the `.container` module.

### CODE REVIEW

File: lib/themes/dosomething/paraneue_dosomething/includes/preprocess.inc [hunk 1]
```diff
@@ -106,23 +91,43 @@ function paraneue_dosomething_preprocess_page(&$vars) {
...
+    $vars['show_sponsors'] = theme_get_setting('show_sponsors');
+  }
+
+  elseif ($type === "static_content" || $type === "fact_page") {
+    /**
+     * Header
+     */
+
+    $header_vars = array(
+      'title' => drupal_get_title(),
```
Comment (angaither) [inline lib/themes/dosomething/paraneue_dosomething/includes/preprocess.inc]: you can get this from `$variables['title']` or something :elephant:

Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/includes/preprocess.inc]: I think I had trouble with that when I was first working on theme functioning the header content...

Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/includes/preprocess.inc]: I'll check again tho

Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/includes/preprocess.inc]: Nope. You be right @angaither ;) Maybe I had issues before when it sat in the `preprocess_page`? Anyhoo, updating it!

**Comment by DFurnes** [comment]
Seems good. :sailboat:

File: lib/themes/dosomething/paraneue_dosomething/templates/static_content/node--static_content.tpl.php [hunk 2]
```diff
@@ -36,19 +53,25 @@
       </div>
     <?php endif; ?>
 
+
     <?php if (!empty($content['field_blocks'])): ?>
     <?php print render($content['field_blocks']); ?>
+      <pre>WHAT THE CRAP ARE THESE</pre>
```
Comment (aaronschachter) [inline lib/themes/dosomething/paraneue_dosomething/templates/static_content/node--static_content.tpl.php:59]: these are the blocks field reference. check out the 11 facts landing page, /facts

Comment (aaronschachter) [inline lib/themes/dosomething/paraneue_dosomething/templates/static_content/node--static_content.tpl.php:59]: because this line will show up there

Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/templates/static_content/node--static_content.tpl.php:59]: Huh, missed that line. :poop: 

@weerd

**Pull request merged**